include/vapaee/dex/tables/markets.hpp

```diff
@@ -1,24 +1,29 @@
 #include "./_aux.hpp"
 
-        
-        // ------------------------------------
-        // TABLE markets
-        // scope: contract
-        TABLE markets_table {
-            uint64_t id;
-            name table;     // xxx.zzz (do not asume it is unique)
-            symbol_code commodity;
-            symbol_code currency;
-            uint64_t primary_key() const { return id; }
-            uint64_t by_table_key() const { return table.value; }
-            uint64_t by_commodity_key() const { return commodity.raw(); }
-            uint64_t by_currency_key() const { return currency.raw(); }
-        };
+#include <vapaee/base/utils.hpp>
+
+
+// scope: contract
+TABLE markets_table {
```

Comment:
Changes to market table:

- Drop `table` field
- Add `uint128_t` secondary index calculated using both commodity and currency raw symbol_code values
- Add function to get pretty string representation of market with format: `{commodity}/{currency}`